### DISCUSSION TITLE
Hi , guys, I am new here, How can I join the Google Summer of Code 2023 CNCF Project Applications ?

### DISCUSSION THREAD

**Discussion by YYDarling** [cncf/mentoring#841] [Q&A] [answered]
Hello everyone, I am totally new here, How can I join the Google Summer of Code 2023 CNCF Project Applications?
Please give me some guidance. 
Thanks for your help.

**Comment by aliok** [comment, marked as answer]
Hi @YunDudali 

Best idea right now is checking the GSoC website to get familiar with the program: https://summerofcode.withgoogle.com/

Also, you might want to read the contributor guide: https://google.github.io/gsocguides/student/

Then you can get yourself familiar with the projects listed in the ideas page: https://github.com/cncf/mentoring/blob/main/summerofcode/2023.md

Also, we have a #mentoring channel in CNCF Slack: https://github.com/cncf/mentoring#communication if you would like assistance on a more specific subject.
